### PR TITLE
Document Backup Accounts

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Backup_Accounts/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: Backup Accounts
+permalink: Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Backup_Accounts
+parent: Accounts and SSO
+grand_parent: Apps and MW
+layout: default
+---
+
+# Backup Accounts
+
+Backup accounts are implemented as [Buteo client plugins](/Reference/Core_Areas_and_APIs/Apps_and_MW/Synchronization/#buteo-client-plugins). The plugin On the Accounts and SSO side, the plugin [provides](../Providers_and_Services#provider-description-files) a [storage service](../Providers_and_Services#predefined-service-types). The service is called `<provider>-backup`, where `<provider>` is the id of the provider.
```
[the remark this answers is not in the document]
Comment (rainemak) [reply Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Backup_Accounts/README.md:11]: I'd put emphasis here that backup accounts are always online accounts. Triggering a backup to an sdcard doesn't require backup account.

**Review by rainemak — CHANGES_REQUESTED** [review]
Minor remarks added. Please give a look.

File: Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Backup_Accounts/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Service configuration
+
+The backup service configuration is done via [Settings UI](../Settings_UI). The location of backup files on the server is configured via settings key `backups_path`. If `OnlineSyncAccountCreationAgent` or `OnlineSyncAccountsSettingsAgent` are used, this setting is configured in the advanced settings dialog.
+
+## Sync profiles
+
+The [sync profiles](/Reference/Core_Areas_and_APIs/Apps_and_MW/Synchronization/#sync-profiles) for the backup accounts are created by the Backup settings UI. There are three profiles for each provider: Backup, BackupQuery and BackupRestore. Each profile matches to an operation, which are described below.
```
[the remark this answers is not in the document]
Comment (vigejolla) [reply Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Backup_Accounts/README.md:19]: > * Backup settings ui is vague definition as same account can be created from Settings -> Accounts and Settings -> System -> Backup

That's what I was referring to. But I guess it's vague if you thought I was referring to something else -> I'll specify more clearly what is meant.

File: Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Backup_Accounts/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vigejolla) [reply Reference/Core_Areas_and_APIs/Apps_and_MW/Accounts_and_SSO/Backup_Accounts/README.md:11]: Removed extra "The plugin", added emphasis.

**Review by rainemak — APPROVED** [review]

**Pull request merged**